src/Emr/Eloquent/PharmacyData.php

```diff
@@ -62,7 +62,6 @@ public function setregisteredStatus($registeredStatus)
 
     public function getCode()
     {
-        // TODO @leo
-        return "This is the Code";
+        return $this->getId();
```

Comment:
The code should return string representation of JSON object. Like this:
`{
    "id": "57e3face1fbe627729dac1ca",
    "name": "Jodie Klein",
    "address": "838 Heyward Street, Ryderwood, Ohio"
  }`